Plugins/Statistics/GitStatistics/LineCounter.cs

```diff
@@ -54,7 +54,7 @@ IEnumerable<FileInfo> GetFiles()
             {
                 foreach (var file in filesToCheck)
                 {
-                    if (extensions.Contains(PathUtil.GetExtension(file)))
+                    if (extensions.Contains(Path.GetExtension(file)))
```

Comment:
Does `HashSet<string>.Contains(null)` thriow?

Reply:
I would assume so - this is a potential issue
(not changed by this PR)

Reply:
I moved the change into the try-catch

Reply:
This has been changed by this PR: `PathUtil.GetExtension` did not return `null` but `""`. So
```suggestion
                    if (extensions.Contains(Path.GetExtension(file) ?? ""))
```
?